app/uk/gov/hmrc/tai/repositories/TaxAccountRepository.scala

```diff
@@ -21,6 +21,7 @@ import play.api.libs.json.JsValue
 import uk.gov.hmrc.domain.Nino
 import uk.gov.hmrc.http.HeaderCarrier
 import uk.gov.hmrc.tai.connectors._
+import uk.gov.hmrc.tai.connectors.cache.Caching
```

Comment:
How did this file know caching before you added this import?

Reply:
from connectors._, this import got added when I moved it to a separate package